docs/source/concepts/Workspace.rst

```diff
@@ -10,131 +10,38 @@ Workspace
-
--  :ref:`TableWorkspace <Table Workspaces>` - A workspace holding data in
-   rows of columns having a particular type (e.g. text, integer, ...).
--  :ref:`WorkspaceGroup <WorkspaceGroup>` - A container for a collection of
-   workspaces. Algorithms given a group as input run sequentially on
-   each member of the group.
-
-Working with Workspaces in Python
----------------------------------
-
-Workspace is an abstract description of an specific workspace implementation. It provides access to a few common properties without any knowledge of what the type of the workspace.
-
-.. _Workspace-Accessing_Workspaces:
-
-Accessing Workspaces
-####################
-
-You can access workspaces using the ``mtd["worskpace_name"]`` command for a specific workspace, or using the ``mtd.ImportAll()`` to create python variables for every workspace in Mantid.  More explanation can be found in `Accessing Workspaces From Python <http://www.mantidproject.org/Accessing_Workspaces_From_Python>`_.
-
-.. testcode:: AccessingWorkspaces
-
-    # This creates a workspace without explicitly capturing the output
-    CreateSampleWorkspace(OutputWorkspace="MyNewWorkspace")
-
-    # You can get a python variable pointing to the workspace with the command
-    myWS = mtd["MyNewWorkspace"]
-    print("The variable myWS now points to the workspace called " + str(myWS))
-
-    # You can also ask Mantid to create matching python variables for all of it's workspaces
-    mtd.importAll()
-    print("MyNewWorkspace has been created that also points to the workspace called " + str(MyNewWorkspace))
-
-    # You can assign a python variable when calling an algorithm and the workspace will match the variable name
-    myOtherWS = CreateSampleWorkspace()
-    print("myOtherWS now points to the workspace called " + str(myOtherWS))
-
-Output:
+Workspaces store data that :ref:`algorithms <algorithm>` operate on. Workspaces are usually stored in-memory. An algorithm can manipulate a workspace in-place or create a new one as an output. 
 
-.. testoutput:: AccessingWorkspaces
-    :options: +NORMALIZE_WHITESPACE
+Workspace is as loose term that encompases a range of possible data structures. All forms of Workspace provide some common information:
 
-    The variable myWS now points to the workspace called MyNewWorkspace
-    MyNewWorkspace has been created that also points to the workspace called MyNewWorkspace
-    myOtherWS now points to the workspace called myOtherWS
+- The type identifier (id) of the workspace (see below), unique to every type of workspace 
+- The name of the workspace, unique to every instance of a workspace even those of the same type
+- The history of :ref:`algorithms <algorithm>` that were run to form and manipulate the workspace to its current state
 
-Workspace Properties
-####################
+.. tip:: In `MantidPlot <http://www.mantidproject.org/MantidPlot:_Help>`__ the data from the workspaces can be graphically viewed, inspected, and plotted in many ways.
```

Comment:
Isn't there a place in the user docs to point people at instead? I'm surprised that it wasn't moved over, but cannot find it.